learn/teams/manage-projects.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Manage Projects
+id: ""
+---
+---
+
+One of the main features of **Team** is the ability to share projects and collaborate on them with other users. Sharing a project with team member is more powerful, as the only users who will be allowed to access the project are users who you have invited to collaborate on the project. You can even choose to restrict what type of access other users have to specific projects.
```

Comment:
Add the importance of having the control here.  Ex: Take some usecase like a person leaving org.
How does it different from having the control on project admin(from Studio page).
To whom he/she can share - only inside the org. how he gets notified etc.